### ISSUE TITLE
Session timeout while pressing "submit" 

### ISSUE THREAD

**Issue by mullhol** [StatCan/wellBeingCheck#240]
I was completing a questionnaire and received the session timeout warning at the same time I pressed submit. This should not be possible. Session timeout warning stayed on the screen as the app returned to the homepage and gave me the thank you toast. 

![image](https://user-images.githubusercontent.com/59370537/80933307-29d7bc00-8d91-11ea-8bfc-76efcade5f80.png)

**Comment by armonkolaei** [comment]
@mullhol @Jiandong-Guo @AJBouzidi we need to know which version is on this test phone. Perhaps its build with 2 min timer and not default 7 min.

**Comment by mullhol** [comment]
newest version of the app on my personal iphone 6, ios 13.3.1
This ticket could be combined with ticket 242.  I submitted this one first as it seemed like a one-off issue, but the more I tested, I realized it was timing out approximately every 7 minutes.  I don't think the time-out reported in this ticket was specific to the screen I was on, I would say it happened about 7 minutes after I downloaded and entered the app.
